cluster_test.go

```diff
@@ -49,3 +52,58 @@ func TestGetLocalSerial(t *testing.T) {
 	println(localSerial)
 
 }
+
+func TestGetClusterConfig(t *testing.T) {
+	config, err := client.GetClusterConfig(defaultCtx)
+	if err != nil {
+		panic(err)
+	}
+	if config.OnefsVersion == nil || config.Timezone == nil {
+		panic("Couldn't get version or time zone")
+	}
+}
+
+func TestGetClusterIdentity(t *testing.T) {
+	identity, err := client.GetClusterIdentity(defaultCtx)
+	if err != nil {
+		panic(err)
+	}
+	log.Debug(defaultCtx, "Get identity name of the cluster '%s'", identity.Name)
+}
+
+func TestGetClusterAcs(t *testing.T) {
+	acs, err := client.GetClusterAcs(defaultCtx)
+	if err != nil {
+		panic(err)
+	}
+	log.Debug(defaultCtx, "%+v", acs)
```

Comment:
better not use log.Debug to print info, please use assert here for checking

Reply:
And other tests also have similar situation, please update

Reply:
Updated